### DISCUSSION TITLE
GitHub Mobile App not displaying some SVGs properly

### DISCUSSION THREAD

**Discussion by knlj** [community/community#60416] [Mobile]
### Select Topic Area

Bug

### Body

Reviving [https://github.com/orgs/community/discussions/4103](https://github.com/orgs/community/discussions/4103) -
Upon updating my profile README today, I went to check it on the github mobile app, but all of the svgs I added appear as [?] icons. 
<img height="400" src="https://github.com/community/community/assets/55034053/127f4aa1-236c-4c70-9983-d59af0fef654">
After doing some digging, it seems this only occurs when closing the `<img>` tags used to display the image (e.g. `<img height="30" width="30" src="static\logos\react.svg"></img>
`). However, this issue does not exist when viewing the README on desktop. I will be removing the closing tags, but I thought this was a bug worth sharing.

**Comment by luigieterovik** [comment]
what about trying "Desktop site"? <br><img src="https://browserstack.wpenginepowered.com/wp-content/uploads/2022/01/Screenshot1.png" height="500px">

**Reply by Arifa100990** [reply]
__
